docs/Understanding-FLINT/FLINT-Inputs/index.rst

```diff
@@ -0,0 +1,18 @@
+FLINT-UI Inputs
+=========================
+
+We currently use five main data-types that are used as FLINT inputs - geoTIFF, JSON, CSV, SQL DB, Shapefile. We have hosted the python notebooks for each data type that contains the instructions and the required dependencies to view each file on a python notebook.
```

Comment:
Replace line 4 with : We currently have five main data types used as FLINT inputs - geoTIFF, JSON, CSV, SQL DB, and Shapefile. Final change and the PR is good to go! 🚀